dts/bindings/gpio/fcs,fxl6408.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2021 Abel Sensors
+# SPDX-License-Identifier: Apache-2.0
+
+description: FXL6408 I2C-based GPIO expander
+
+compatible: "fcs,fxl6408"
```

Comment:
Curious where `fcs` is coming from?

Reply:
I found it in /zephyr/dts/bindings/vendor-prefixes.txt. It stands for the vendor Fairchild Semiconductor.